apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -391,7 +391,7 @@ export default function teacherSections(state=initialState, action) {
     const prevSections = action.reset ? [] : state.sections;
     return {
       ...state,
-      sectionIds: prevSectionIds.concat(sections.map(section => section.id)),
+      sectionIds: _.uniq(prevSectionIds.concat(sections.map(section => section.id))),
```

Comment:
With the removal of the duplicated call to setSections in our story, this is actually no longer necessary but still seems like goodness. Thoughts?